src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -18,6 +18,48 @@ export interface NameRequestFilingIF {
   correctNameOption?: CorrectNameOptions // only used by UI for save and resume
 }
 
+/** Interface for amalgamation filing data saved to the Legal API. */
+export interface AmalgamationFilingIF {
```

Comment:
This is super helpful for my ticket 🙌

Reply:
There _will_ be new fields in here, but we're not there yet. Anyways, you only need to create a basic draft. You might not even declare an IF in auth-web...?

Reply:
Yes, that's what I was thinking. I'm going to see once I reach that stage. I'm going to create a draft with minimal data and see how that goes.

Reply:
Just do what's done for IAs.